module.jai

```diff
@@ -183,26 +185,29 @@ get_last_query_result :: (conn: *PGconn) -> *PGresult {
     return query_result;
 }
 
-check_query_result :: (query_res: *PGresult) -> has_results: bool, success: bool {
+check_query_result :: (query_res: *PGresult) -> has_results: bool, updated_rows: int, success: bool {
```

Comment:
`update_rows` should be the third return value here for the same reason as why you made it the second value for `execute`